README.md

```diff
@@ -1,4 +1,4 @@
-<img src="docs/kubescape.png" width="300" alt="logo" align="center">
+<a href="https://cloud.armosec.io/account/login?utm_campaign=Community%20Classroom%20%7C%20Kunal%20Kushwaha&utm_source=Influencers%20&utm_medium=Community%20Classroom%20&utm_content=Kubescape%20registration%20"><img src="docs/kubescape.png" width="300" alt="logo" align="center"></a>
```

Comment:
You can include this URL in place of that long URL it works the same 
`https://cloud.armosec.io/`

Reply:
I will update it, thank you .